src/components/ProfitAndLossChart/ProfitAndLossChart.tsx

```diff
@@ -262,99 +164,21 @@ export const ProfitAndLossChart = ({
-    expenses: -(pnl?.totalExpenses || 0),
-    expensesUncategorized: -(pnl?.uncategorizedOutflows || 0),
-    totalExpensesInverse: pnl?.totalExpensesInverse || 0,
-    uncategorizedOutflowsInverse: pnl?.uncategorizedOutflowsInverse || 0,
-    netProfit: pnl?.netProfit || 0,
-    selected:
-      !!pnl
-      && pnl.month === selectionMonth.month + 1
-      && pnl.year === selectionMonth.year,
-    year: pnl?.year,
-    month: pnl?.month,
-    base: 0,
-    loading: pnl?.isLoading ? loadingValue : 0,
-    loadingExpenses: pnl?.isLoading ? -loadingValue : 0,
-  })
-
-  const theData = useMemo(() => {
-    if (loaded !== 'complete' || (loaded === 'complete' && !anyData)) {
-      const loadingData = []
-      const today = Date.now()
-      for (let i = 11; i >= 0; i--) {
-        const currentDate = sub(today, { months: i })
-        loadingData.push({
-          name: format(currentDate, compactView ? 'LLLLL' : 'LLL'),
-          revenue: 0,
-          revenueUncategorized: 0,
-          totalExpensesInverse: 0,
-          uncategorizedOutflowsInverse: 0,
-          expenses: 0,
-          expensesUncategorized: 0,
-          netProfit: 0,
-          selected: false,
-          year: currentDate.getFullYear(),
-          month: currentDate.getMonth() + 1,
-          loading: 1000 * Math.pow(-1, i + 1) * (((i + 1) % 12) + 1) + 90000,
-          loadingExpenses:
-            -1000 * Math.pow(-1, i + 1) * (((i + 1) % 2) + 1) - 90000,
-          base: 0,
-        })
-      }
-      return loadingData
-    }
-
-    return data
-      ?.map(x => {
-        const totalExpenses = x.totalExpenses || 0
-        if (totalExpenses < 0 || x.uncategorizedOutflows < 0) {
-          return {
-            ...x,
-            totalExpenses: totalExpenses < 0 ? 0 : totalExpenses,
-            uncategorizedOutflows:
-              x.uncategorizedOutflows < 0 ? 0 : x.uncategorizedOutflows,
-            totalExpensesInverse: totalExpenses < 0 ? -totalExpenses : 0,
-            uncategorizedOutflowsInverse:
-              x.uncategorizedOutflows < 0 ? -x.uncategorizedOutflows : 0,
-          }
-        }
-
-        return x
-      })
-      ?.filter(
-        x =>
-          differenceInMonths(
-            startOfMonth(new Date(x.year, x.month - 1, 1)),
-            chartWindow.start,
-          ) >= 0
-          && differenceInMonths(
-            startOfMonth(new Date(x.year, x.month - 1, 1)),
-            chartWindow.start,
-          ) < 12
-          && differenceInMonths(
-            chartWindow.end,
-            startOfMonth(new Date(x.year, x.month - 1, 1)),
-          ) >= 0
-          && differenceInMonths(
-            chartWindow.end,
-            startOfMonth(new Date(x.year, x.month - 1, 1)),
-          ) <= 12,
-      )
-      .map(x => summarizePnL(x))
-  }, [selectionMonth, chartWindow, data, loaded, compactView])
+  /** @TODO temp */
+  const len = period === 'year' ? 1 : period === 'quarter' ? 4 : 12
```

Comment:
FYI: Temporary - just to limit number of bars on the chart. It needs to be removed when API endpoint is ready